pwndbg/commands/addressify.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+
+from typing import Any
+
+import gdb
+
+import pwndbg.color.message as M
+import pwndbg.commands
+
+
+class AddressifyFunction(gdb.Function):
+    # GDB standalone function to convert a hex string to little-endian address and return the address.
+
+    def __init__(self) -> None:
+        super().__init__("addressify")
+
+    def invoke(self, *args: gdb.Value) -> gdb.Value:
+        if not args or len(args) != 1:
+            raise gdb.GdbError("addressify expects a single hex string argument.")
+
+        arg = args[0].string()
+        address = addressify_common(arg)
+        return gdb.Value(address)
+
+
+AddressifyFunction()
+
+
+@pwndbg.commands.Command
+@pwndbg.commands.OnlyWhenRunning
+def addressify(*args: Any) -> None:
+    # Pwndbg command to convert hex string to little-endian address and print the result.
+    if len(args) == 1 and args[0] in ("-h", "--help"):
```

Comment:
This should be handled by `ArgparsedCommand`; we don't want any command to be implemented with the `Command` decorator fwiw